code/vehicle.h

```diff
@@ -12,6 +12,7 @@ typedef struct VehicleStruct {
   uint16_t velocity;                   ///< [km/h] Vehicle velocity
   uint16_t angleAccPedal;              ///< [%] Vehicle throttle pedal pressed percentage 
   uint16_t angleBrakePedal;            ///< [%] Vehicle brake pedal pressed percentage
+  uint16_t vehicleReport;              ///< Time to report Vehicle status [S]
```

Comment:
Suggestion: replace with `time_t` from `time.h`. This is usually the standard type used to store time in C.

```suggestion
  time_t time;    ///<  [s] timestamp from last Vehicle status report
```

ref: https://en.cppreference.com/w/c/chrono/time_t